mysql-plugin/src/main/java/io/cdap/plugin/mysql/MysqlConnectorConfig.java

```diff
@@ -58,6 +60,8 @@ public Properties getConnectionArgumentsProperties() {
     prop.put(JDBC_PROPERTY_CONNECT_TIMEOUT, "20000");
     prop.put(JDBC_PROPERTY_SOCKET_TIMEOUT, "20000");
     prop.put(JDBC_REWRITE_BATCHED_STATEMENTS, "true");
+    // MySQL property to ensure that TINYINT(1) type data is not converted to MySQL Bit/Boolean type in the ResultSet.
+    prop.put(MYSQL_TINYINT1_IS_BIT, "false");
```

Comment:
Does it makes sense to use `putIfAbsent()` so that if user has provided it in connection args in plugin config we don't end up overriding it?